sfa_dash/blueprints/aggregates.py

```diff
@@ -331,8 +337,60 @@ def get(self, uuid):
                     observation.update(obs)
                     self.observation_list.append(observation)
             self.insert_plot(uuid, start, end)
-            self.set_template_args()
-        return render_template(self.template, **self.template_args())
+            self.set_template_args(start, end, **kwargs)
+        return render_template(self.template, **self.temp_args)
+
+    def format_params(self, form_data):
+        start_dt = pd.Timestamp(form_data['start'], tz='utc')
+        end_dt = pd.Timestamp(form_data['end'], tz='utc')
+        params = {
+            'start': start_dt.isoformat(),
+            'end': end_dt.isoformat(),
+        }
+        headers = {'Accept': form_data['format']}
+        return headers, params
+
+    def post(self, uuid):
+        form_data = request.form
```

Comment:
can format_params and post be shared with main.py?

Reply:
I think that would be ideal. The only obstacle I see is handling the routing, because right now the forms are set to just post too whatever the current path is, but that may be simple to handle.